glslc/src/main.cc

```diff
@@ -133,8 +141,12 @@ int main(int argc, char** argv) {
       compiler.SetWorkingDirectory(workdir.str());
     } else if (arg.starts_with("-fshader-stage=")) {
       const string_piece stage = arg.substr(std::strlen("-fshader-stage="));
-      force_shader_stage = glslc::GetShaderStageFromCmdLine(arg);
-      if (force_shader_stage == EShLangCount) {
+      // forced_shader_stage can be 1) one of the forced shader kinds, which
+      // means following files will be compiled with the specified shader
+      // kind; 2) shaderc_glsl_infer_from_source, which means 'arg' doesn't
+      // specify a valid shader kind, an error should be emitted for this.
```

Comment:
The above really should be a part of `GetForcedShaderKindFromCmdLine`'s blurb, instead of existing at every call site.